lib/Form/Templates/InitialForm.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace MailPoet\Form\Templates;
```

Comment:
Should we move the actual templates into a separate directory? When we have 50 templates it will be difficult to find the other classes in this directory.